R/get-data.R

```diff
@@ -32,28 +35,33 @@
 #'
 #' @export
 get_data_frame <- function(
-    data_frame,
-    format = c("glimpse", "print", "json"),
-    dims = c(5, 100)
+  data_frame,
+  format = c("skim", "glimpse", "print", "json"),
```

Comment:
I made it the default because it's otherwise a little tricky to prompt the model to use the skim format (by saying "skim the dataset"). It might make sense to have a global option in the future (or something available from `btw()`).

Anyway, we might want to roll back before merging

Reply:
Yup, all good from my perspective.

If folks want a different format interactively, they can `btw(get_data_frame(df, format = ...))`